pytroll_product_filter/__init__.py

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env python3
+"""Package file for the product-filter."""
+from importlib.metadata import version
+
+try:
+    __version__ = version(__name__)
+except ModuleNotFoundError:
+    __version__ = "?"
+PACKAGE_NAME = __name__
```

Comment:
How is this compared to what I am used to concerning getting the version from git?

This is what I usually do (in my package `__init__.py` file to get the version automatically using git
```
from pkg_resources import DistributionNotFound, get_distribution

try:
    __version__ = get_distribution(__name__).version
except DistributionNotFound:
    # package is not installed
    pass
```

Reply:
Hi!

The versioning is very similar and is performed automatically by the [`poetry-dynamic-versioning`](https://github.com/mtkennerly/poetry-dynamic-versioning) plugin, which I added as a system requirement in the 22c4b354114b6bafc0c6aaad23fe6405f0871f41 commit.

In a nutshell, that plugin dynamically modifies the `version` placeholder present in the `pyproject.toml` file when you produce a release. The generated versions follow the same behaviour as what you had before here in `pytroll-product-filter`.

Since I'm using the `poetry` engine now, you'd need to use `poetry` to publish the package instead of `twine`. For instance, 

```
poetry publish --build --repository satsa-gitlab
```

or, if you want to publish to a local repository [configured](https://python-poetry.org/docs/repositories/#publishing-to-a-private-repository) as `my-local-repo`,
```
poetry publish --build --repository my-local-repo
```

Reply:
Oh, I see, thanks, then I am fine.